packages/vaadin-template-renderer/src/vaadin-template-renderer-templatizer.js

```diff
@@ -12,51 +12,81 @@ export class Templatizer extends PolymerElement {
     return 'vaadin-template-renderer-templatizer';
   }
 
-  static get properties() {
-    return {
-      __template: Object,
-      __TemplateClass: Object,
-      __templateInstance: Object
-    };
-  }
+  __template = null;
+  __templateInstances = new Set();
```

Comment:
We are not currently using [class fields](https://caniuse.com/mdn-javascript_classes_public_class_fields) as we still support Safari 13.

So please initialize these in the constructor:

```suggestion
  constructor() {
    super();
    this.__template = null;
    this.__templateInstances = new Set();
  }
```

Reply:
Yeah, thanks! Maybe there is a list of browsers somewhere that are intended to support?

Reply:
Good point. Created an issue to clarify the supported browsers: https://github.com/vaadin/web-components/issues/319